blog/content/introtogonum.md

```diff
@@ -0,0 +1,160 @@
+
+We accept this cost because we believe it comes with an enormous benefit.
+The Gonum function and method signatures make the algorithmic outputs clear,
+and it is easy to identify the code being executed and examine the implementation.
+Science is built on understanding how an experiment was performed and
+what the results are.
+We believe scientific code should be no different.
+In fact, Gonum is partly distinguished from other libraries in that you only need
+one programming language to read Gonum code (that said, it is also easy to call
+into existing C and Fortran libraries).
+This philosophy of simplicity has many follow-on benefits.
+For example, dense one-liners can be hard to understand, creating a barrier for 
+others to confirm correctness, and making future modifications difficult.
+Additionally, one-liners often obscure the necessary operations to execute that
+line of code.
+This creates a tension between code that is terse, and a targeted implementation
+that is efficient.
+The construction of Gonum encourages users to implement algorithms in a uniform
+style, making the code easy to read and consistent between users.
+
+The use of Go as a base language has many benefits beyond simplicity.
+Go is fast, approaching the speed of C.
+Go's language primitives are great for parallel computing, and in fact Gonum
+code can often be made to run in parallel by setting a flag.
+Go is designed for ["programming in the large"](https://talks.golang.org/2012/splash.article),
+a huge benefit for building an ecosystem of libraries that work with and build
+on top of one another.
+Go has extremely legible compile errors, full stack traces, trivial package downloading,
+automatic makefiles and import resolution, code formatting tools, easy code documentation,
+a race detector, and the list goes on.
+
+We encourage you to try Go and Gonum if
+- You are tired of sluggish performance, and with fighting vectorization and C
+- You are struggling with managing programs as they grow larger and/or
+- You struggle to re-use even the code you tried to make reusable
+- You would like easy access to parallel computing
+- You want code to be fully transparent, and you want the ability to read the source code you use
+- You'd like a compiler to catch mistakes early, but hate fighting linker errors and unintelligible compile errors
+
+We believe in a new foundation for scientific computing, and we hope you'll join us.
+Please check back for more discussion on the philosophy of Gonum and the design
+decisions for specific packages.
+
+## Getting started with Gonum
+### Downloading the libraries
+First, you need to [install Go](https://golang.org/doc/install) if it is not
+already installed on your machine.
+It is recommended that you use the most recent version of Go.
+Run 
+```
+go get -u -t gonum.org/v1/gonum/...
+```
+to download the main set of Gonum packages.
+Then run
+```
+go get -u -t gonum.org/v1/plot
```

Comment:
This isn't necessary, because the first line downloades the gonum repository. I had `plot` there because it is not part of the gonum repository. We can add it in when we're stabilized plot.